packages/main/test/pages/Calendar.html

```diff
@@ -69,6 +70,11 @@
 		<ui5-calendar id="calendar4" data-ui5-compact-size min-date="7/7/2020" max-date="20/10/2020" timestamp="1594080000" format-pattern="dd/MM/yyyy"></ui5-calendar>
 	</section>
 
+	<section>
+		<ui5-title>Calendar with two type</ui5-title>
```

Comment:
Calendar with primary and secondary calendar type

Reply:
Done